week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,14 +10,18 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(arr) {
+  let pink = arr.every(name => name.includes("pink"));
```

Comment:
This looks mostly right, but `includes` is an interesting check here... What would happen if our array was:

`["pink", "not pink", "pink"]`

?